share/spice/quixey/quixey.js

```diff
@@ -123,7 +123,7 @@
                     img_m:         screenshot,
                     title:         item.name,
                     heading:       item.name,
-                    rating:        item.rating,
+                    rating:        features.rating_avg || item.rating,
```

Comment:
I couldn't find any documentation on the API, but I did find that this attribute has the expected value for all the cases I tried (instead of the 0.x sometimes found in `item.rating`. Any particular reason to prefer `custom.avg_rating` instead?